modules/components/tag-input.ts

```diff
@@ -352,6 +360,16 @@ export class TagInputComponent extends TagInputAccessor implements OnInit {
      */
     public inputTextValue = '';
 
+
+    /**
+     * @desc removes the tab index if it is set - it will be passed through to the input
+     * @name tabindexAttr
+     * @type {string}
+     */
+    @HostBinding('attr.tabindex') get tabindexAttr(): string {
```

Comment:
would it be possible to put ` get tabindexAttr(): string ` on the line below and add the modifier `public`? just to be consistent with the rest